x-pack/plugin/security/src/test/java/org/elasticsearch/test/SecuritySettingsSource.java

```diff
@@ -403,7 +403,7 @@ public static class UnregisteredSecuritySettingsPlugin extends Plugin {
         );
         public static final Setting<Boolean> CLUSTER_STATE_ROLE_MAPPINGS_ENABLED = Setting.boolSetting(
             "xpack.security.authc.cluster_state_role_mappings.enabled",
-            false,
+            true,
```

Comment:
I think this should work but in case some test somewhere makes an explicit assumption this is `false` I'm fine with keeping it `false`

Reply:
If we remove this from serverless after this PR is merged we can come back and remove this property. Setting it to true here makes sense I think.